static/modules/gamemanager.js

```diff
@@ -95,6 +97,13 @@ class GameManager {
     }
   }
 
+  addDisplayToDiv(id) {
+    const displayDiv = document.getElementById(id);
+    if (displayDiv != undefined) {
+      displayDiv.appendChild(this.displayCanvas);
+    }
+  }
+
```

Comment:
Hmm. Why did this get added? This got moved over to displaymanager I'm pretty sure...
It also has literally no references but itself so just get rid of it.

Reply:
I don't recall how this got added, will be removed.